src/actions/link.actions.test.js

```diff
@@ -1,175 +1,176 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
@talend-glorieux i mean the configuration merged in Talend/tools#15 will not work until eslint plugin is released